functions/Get-DbaSsisEnvironmentVariable.ps1

```diff
@@ -0,0 +1,281 @@
+                                    value			= ev.value,
+                                    ev.sensitive_value,
+                                    ev.base_data_type,
+                                    decrypted		= decrypted.value
+                                FROM internal.environment_variables ev
+
+                                    CROSS APPLY (
+                                        SELECT
+                                            value	= CASE base_data_type
+                                                        WHEN 'nvarchar' THEN CONVERT(NVARCHAR(MAX), DECRYPTBYKEY(sensitive_value))
+                                                        WHEN 'bit' THEN CONVERT(NVARCHAR(MAX), CONVERT(bit, DECRYPTBYKEY(sensitive_value)))
+                                                        WHEN 'datetime' THEN CONVERT(NVARCHAR(MAX), CONVERT(datetime2(0), DECRYPTBYKEY(sensitive_value)))
+                                                        WHEN 'single' THEN CONVERT(NVARCHAR(MAX), CONVERT(DECIMAL(38, 18), DECRYPTBYKEY(sensitive_value)))
+                                                        WHEN 'float' THEN CONVERT(NVARCHAR(MAX), CONVERT(DECIMAL(38, 18), DECRYPTBYKEY(sensitive_value)))
+                                                        WHEN 'decimal' THEN CONVERT(NVARCHAR(MAX), CONVERT(DECIMAL(38, 18), DECRYPTBYKEY(sensitive_value)))
+                                                        WHEN 'tinyint' THEN CONVERT(NVARCHAR(MAX), CONVERT(tinyint, DECRYPTBYKEY(sensitive_value)))
+                                                        WHEN 'smallint' THEN CONVERT(NVARCHAR(MAX), CONVERT(smallint, DECRYPTBYKEY(sensitive_value)))
+                                                        WHEN 'int' THEN CONVERT(NVARCHAR(MAX), CONVERT(INT, DECRYPTBYKEY(sensitive_value)))
+                                                        WHEN 'bigint' THEN CONVERT(NVARCHAR(MAX), CONVERT(bigint, DECRYPTBYKEY(sensitive_value)))
+                                                    END
+                                    ) decrypted
+
+                                WHERE environment_id = $($e.EnvironmentId);
+
+                                CLOSE SYMMETRIC KEY $encKey;
+"@
+
+                            $ssisVariables = Invoke-DbaSqlCmd -ServerInstance $instance -Database SSISDB -Query $sql -As DataTable
+                            
+                            foreach($variable in $ssisVariables)
+                            {
+                                if($variable.sensitive -eq $true)
+                                {
+                                    $value = $variable.decrypted
+                                }
+                                else
+                                {
+                                    $value = $variable.value
+                                }
+
+                                [PSCustomObject]@{
+                                    Server          = $instance
```

Comment:
Can you add 
- ComputerName
- InstanceName
- SqlInstance
instead of Server, please?
Those are our standards. You'll find them in most Get- functions.